.github/workflows/continuous_integration.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - master
       - release
-      - 'v3.13-dev'
+      - 'v3'
```

Comment:
Our `release` branch is pointing to `v3.13.2`. Should that be moved or deleted?

Reply:
The release branch is not in direct parent path behind v3, so I would prefer to just skip it.

Reply:
Deleted the rule now, and also deleted the branch itself